### PR TITLE
Add Read inventory

### PR DESCRIPTION
**Pull request by zhengtr**
- Added endpoint to read-inventory in `routes.py`
- Added test cases for read-inventory

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -24,6 +24,25 @@ def index():
     )
 
 
+######################################################################
+# RETRIEVE A PET
```
Comment (rachitjain2706) [inline service/routes.py:28]: Update comment here

File: tests/test_routes.py [hunk 2]
```diff
@@ -13,12 +13,14 @@
 from service.models import db, init_db, Inventory, Condition
 from tests.factories import InventoryFactory
 from service.common import status  # HTTP Status Codes
+from tests.factories import InventoryFactory
```
Comment (rachitjain2706) [inline tests/test_routes.py:16]: Import statement has been added twice.

File: tests/test_routes.py [hunk 3]
```diff
@@ -124,4 +146,4 @@ def test_create_inventory_bad_quantity(self):
         test_item = item.serialize()
         test_item["quantity"] = "male"    # wrong case
         response = self.client.post(BASE_URL, json=test_item)
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```
Comment (rachitjain2706) [inline tests/test_routes.py:149]: An extra line should be there at the end of the file.

**Review by rachitjain2706 — APPROVED** [review]
LGTM.

File: service/routes.py [hunk 4]
```diff
@@ -24,6 +24,25 @@ def index():
...
+    Retrieve a single Inventory
+
+    This endpoint will return a Inventory based on it's id
+    """
+    app.logger.info("Request for pet with id: %s", inventory_id)
+    inventory = Inventory.find(inventory_id)
+    if not inventory:
+        abort(status.HTTP_404_NOT_FOUND, f"Inventory with id '{inventory_id}' was not found.")
+
+    app.logger.info("Returning pet: %s", inventory.name)
```
Comment (nishanthsanjeev) [inline service/routes.py:42]: Need to fix logger info. Should be "item" instead of "pet".

File: service/routes.py [hunk 5]
```diff
@@ -24,6 +24,25 @@ def index():
...
+# RETRIEVE AN INVENTORY ITEM
+######################################################################
+@app.route("/inventory/<int:inventory_id>", methods=["GET"])
+def get_inventory(inventory_id):
+    """
+    Retrieve a single Inventory
+
+    This endpoint will return a Inventory based on it's id
+    """
+    app.logger.info("Request for pet with id: %s", inventory_id)
```
Comment (nishanthsanjeev) [inline service/routes.py:37]: Need to fix logger info. Should be "item" instead of "pet".

**Review by nishanthsanjeev — CHANGES_REQUESTED** [review]

**Comment by sahildeshpande25** [comment]
@zhengtr PRs shouldn't have an enhancement label. 
https://nyu-devops-spring23-3.slack.com/archives/C04NWAD335G/p1677771021460519

**Comment by zhengtr** [comment]
> @zhengtr PRs [shouldn't have an enhancement label](https://nyu-devops-spring23-3.slack.com/archives/C04NWAD335G/p1677771021460519).

Got it, thanks for letting me know.

**Pull request merged**